app/assets/javascripts/discourse/app/templates/exception.gjs

```diff
@@ -3,47 +3,58 @@ import RouteTemplate from "ember-route-template";
 import { and } from "truth-helpers";
 import ConditionalLoadingSpinner from "discourse/components/conditional-loading-spinner";
 import DButton from "discourse/components/d-button";
+import PluginOutlet from "discourse/components/plugin-outlet";
 import icon from "discourse/helpers/d-icon";
+import lazyHash from "discourse/helpers/lazy-hash";
 import { i18n } from "discourse-i18n";
 
 export default RouteTemplate(
   <template>
-    <div class="container">
-      {{#if (and @controller.errorHtml @controller.isForbidden)}}
-        <div class="not-found">{{htmlSafe @controller.errorHtml}}</div>
-      {{else}}
-        <div class="error-page">
-          <div class="face">:(</div>
-          <div class="reason">{{@controller.reason}}</div>
-          {{#if @controller.requestUrl}}
-            <div class="url">
-              {{i18n "errors.prev_page"}}
-              <a
-                href={{@controller.requestUrl}}
-                data-auto-route="true"
-              >{{@controller.requestUrl}}</a>
-            </div>
-          {{/if}}
-          <div class="desc">
-            {{#if @controller.networkFixed}}
-              {{icon "circle-check"}}
+    <PluginOutlet
+      @name="exception-wrapper"
+      @outletArgs={{lazyHash
+        controller=@controller
```

Comment:
passing in a whole controller into an outlet is an anti-pattern, it's a much better api to provide it with specific set of properties and actions.